src/Components/Subscriptions/Table/index.jsx

```diff
@@ -71,78 +78,88 @@ const Table = ({ data }) => {
   };
 
   const closeModal = () => {
+    setShowConfirmDeleteModal(false);
     setShowDeleteModal(false);
+    history.goBack();
   };
 
   return (
-    <div>
-      <table className={styles.tableContainer}>
-        <thead>
-          <tr>
-            <th>Classes</th>
-            <th>Members</th>
-            <th>Date</th>
-          </tr>
-        </thead>
-        <tbody>
-          {tableData.length > 0 ? (
-            tableData.map((subscription) => (
-              <tr className={styles.line} key={subscription._id}>
-                <td
-                  className={`${styles.item} ${styles.td}`}
-                >{`${subscription.classes.day} ${subscription.classes.time}`}</td>
-                {!subscription.members ? (
-                  <td className={`${styles.item} ${styles.td}`}>{'empty'}</td>
-                ) : (
+    <Router>
```

Comment:
This Router doesn't work here, i've been trying to solve it, and it's easier to just take it out and import {Link} from react-router-dom